.github/workflows/light-system-programs-tests.yml

```
@@ -6,39 +6,43 @@ on:
       - "zk.js/**"
       - "programs/**"
       - "light-circuits/**"
-      - ".github/workflows/system-programs-tests.yml"
+      - "psp-examples/**"
+      - ".github/workflows/light-system-programs-tests.yml"
   pull_request:
     branches:
       - main
     paths:
       - "zk.js/**"
       - "programs/**"
       - "light-circuits/**"
-      - ".github/workflows/system-programs-tests.yml"
+      - ".github/workflows/light-system-programs-tests.yml"
+      - "psp-examples/**"
     types:
       - opened
       - synchronize
       - reopened
       - ready_for_review
 
-name: system-programs-tests
+name: system-programs-psp-examples-tests
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   system-programs:
-    name: system-programs
+    name: system-programs-psp-examples
     if: github.event.pull_request.draft == false
     runs-on: buildjet-16vcpu-ubuntu-2204
     strategy:
       matrix:
         include:
-          - test: system-programs-tests-transaction-user-verifiers-provider
-            sub-tests: '["sp-functional", "sp-user", "sp-verifiers", "sp-provider"]'
-          - test: system-programs-tests-user-merge
-            sub-tests: '["sp-user-merge", "zk.js"]'
+          - test: system-programs-tests
```

```suggestion
          - test: system-programs-psp-examples
```

I think would make more sense